README.md

```diff
@@ -120,7 +120,10 @@ let app = new EmberApp(defaults, {
 
 *Ember***Data** uses `UUID V4` by default to generate identifiers for new data created on the client. Identifier generation is configurable, but we also for convenience will polyfill
 the necessary feature if your browser support or deployment environment demands it. To
-activate this polyfill:
+activate this polyfill;
+* install `@embroider/macros` if not already installed
+* install `@ember-data/store` if not already installed
+* add this to your ember-cli-build.js file:
```

Comment:
we should align the config to the others so that we don't have to make folks do this. For all the other configs we just copy-forward into the appropriate packages from the `emberData` config, including setting up macros configs when needed.

Reply:
is that something likely to happen soon? Is it worth not having clear instructions until then?

Reply:
Open an issue its something I'll get done for 4.12.x + 5.2+

Reply:
Thanks a bunch - done https://github.com/emberjs/data/issues/8675